src/pages/auth/components/notification.js

```diff
@@ -0,0 +1,21 @@
+import 'react-toastify/dist/ReactToastify.css'
+import { toast } from 'react-toastify'
+import { en } from '../../../localization/strings/en'
+
+export const loginError = () => {
```

Comment:
better to receive error message from BE and show it here. so no need to create custom messages for errors and write handleReaquestError function instead of login + register error

Reply:
done